fe1-web/src/features/digital-cash/hooks/DigitalCashHooks.ts

```diff
@@ -23,26 +23,32 @@ export namespace DigitalCashHooks {
   };
 
   /**
-   * Gets the current lao id
+   * Gets the current lao id, throws an error if there is none
    */
-  export const useCurrentLaoId = () => useDigitalCashContext().useCurrentLaoId();
+  export const useAssertCurrentLaoId = () => useDigitalCashContext().useAssertCurrentLaoId();
```

Comment:
I seem to recall we discussed a lot about the naming of `useAssertCurrentLaoId`, I'm still not convinced, but I'm very frustrated that I can't think of a better name ... the only alternative I have in mind is `useCurrentLaoIdUnsafe`😅

Reply:
Haha yeah me neither 😂 Honestly I think `useCurrentLaoIdUnsafe` is not much better as `unsafe` usually implies that you should not use it whereas I would argue in this case we should actually use it (almost) everywhere. Maybe it is even possible to actually use it everywhere now given that we have a clear separation of screens there we want to assert the existence of a current lao and screens where it will always be undefined. The error thrown by the function is caught be the error boundary that then makes the application navigate to the home screen.

Reply:
If we use it everywhere we could even rename it to just `useCurrentLaoId` and remove the old function named like this?

Reply:
Yes, since my comment above, my thinking has evolved exactly along those lines. 😁

Reply:
👍😁

Reply:
Also separate PR I assume 😂